gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -261,12 +263,14 @@ public Builder setExecutorProvider(ExecutorProvider executorProvider) {
      * Sets the CredentialsProvider which will acquire the credentials for making calls to the
      * service. Credentials will not be acquired until they are required.
      */
+    @Deprecated
     public Builder setCredentialsProvider(CredentialsProvider credentialsProvider) {
```

Comment:
Yes it does. Merging this won't break GCJ; you'll just be greeted with a bunch of deprecation warnings.